templates/quiz/hx-question.html

```diff
@@ -0,0 +1,103 @@
+            '<rect class="fill-blue-800" x="7.03" y="6.49" width="37" height="37"></rect><path style="fill: ghostwhite;" d="M18.97,21.07l1.22-5.05a18.39,18.39,0,0,1,6.63-1.28c3.6,0,6.23,1.45,6.23,4.85,0,2.17-1.08,4.19-4.15,5.44l-.03.13c2.70.59,3.86,2.24,3.86,4.85,0,4.55-3.53,6.9-8.21,6.9-4.65,0-7.32-2.27-7-5.90l3.63-1.08c-.16,1.51.85,3.23,3.4,3.23a3.26,3.26,0,0,0,3.53-3.43c0-2.08-1.55-2.77-4.62-2.90l.3-2.9c4.48,0,4.55-2.77,4.55-3.2,0-1.41-.72-2.21-2.54-2.21a7.06,7.06,0,0,0-2.73.5l-.76,2.44Z"></path>',
+            '<circle class="fill-purple-800" cx="24.55" cy="25.02" r="20.5"></circle><path style="fill: ghostwhite;" d="M24.06,36.83l.59-5.64H15.35l-.03-3.66,7.62-12.17h7.69L29.34,27.52h3.23l-.72,3.66H28.98l-.59,5.64Zm1.84-17.72H24.4l-5.04,8.41h5.67Z"></path>',
+            '<polygon class="fill-pink-800" points="25.24 2.93 3.42 18.78 11.76 44.42 38.72 44.42 47.05 18.78 25.24 2.93"></polygon><path style="fill: ghostwhite;" d="M23.97,36.87c-4.38,0-6.97-2.04-6.65-5.72l3.52-1.05c-.16,1.47.73,3.13,3.2,3.13a3.23,3.23,0,0,0,3.51-3.42c0-2.43-2.07-2.94-4.44-2.94-1.02,0-2.08.09-3.00.16l-1.21-1.31L19.94,15.69H32.48l-.03,5.85-3.58.32-.31-2.52h-5.12l-.41,4.06c.38-.03.8-.03,1.18-.03,4.38,0,7.93,1.18,7.93,6.24C32.13,34.7,28.51,36.87,23.97,36.87Z"></path>',
+            '<polygon class="fill-orange-800" points="25.03 1.95 0.78 43.95 49.28 43.95 25.03 1.95"></polygon><path style="fill: ghostwhite;" d="M24.01,36.97c-4.41,0-6.45-3.05-6.45-8.33,0-5.31,2.07-11.76,8.78-11.76,2.19,0,4.47.69,5.61,2.7l-2.48,2.31a4.00,4.00,0,0,0-3.36-1.59c-2.58,0-3.80,1.98-4.26,6.21a5.46,5.46,0,0,1,4.71-2.43c3.53,0,4.8,2.64,4.8,5.46A7.06,7.06,0,0,1,24.01,36.97Zm.9-9.62a3.03,3.03,0,0,0-3.21,2.7c-.03,2.31.57,3.39,2.49,3.39,2.22,0,3.09-1.5,3.09-3.48C27.28,28.3,26.68,27.34,24.91,27.34Z"></path>',
+        ) %}
+        {% set container_colors = cycler(
+            'bg-green-300 border-green-600',
+            'bg-yellow-300 border-yellow-600',
+            'bg-blue-300 border-blue-600',
+            'bg-purple-300 border-purple-600',
+            'bg-pink-300 border-pink-600',
+            'bg-orange-300 border-orange-600',
+        ) %}
+
+        <div class="grid grid-cols-1 md:grid-cols-2 gap-x-12 gap-y-16 mb-8 w-full max-w-screen-lg mx-auto">
+            {% for choice in question.choices %}
+            <div>
+                <div class="turn-pre-into-ace relative py-4 px-1 border-4 rounded-lg shadow-lg
+                    {{container_colors.next()}}
+                    {% if choice.number == progress.last_wrong_answer %}incorrect-option{% endif %}
+                    transition
+                    "
+                    _="on click if I do not match .incorrect-option then
```

Comment:
Is this a language specific to HTMX?

Reply:
It's a separate language made by the same people as HTMX called hyperscript.org

HTMX is focused on handling user interaction by calling a server. Hyperscript focuses on in-browser interaction to fill in the gaps where a server request wouldn't work or wouldn't make sense